src/content/test/semantics/table-semantics.tsx

```diff
@@ -4,7 +4,7 @@ import { createWithTitle, React } from '../../common';
 
 export const infoAndExamples = createWithTitle(
     {
-        pageTitle: 'Data tables',
+        pageTitle: 'Table semantics',
```

Comment:
The requirement name was [recently changed](https://github.com/microsoft/accessibility-insights-web/commit/6957c1a241d8c59de3a2f95ddfbf1808efff1a18)